File: user/Domain/EventModels/PaymentInfoSentEvent.cs
using System;
using PTManagementSystem.Domain.Entities;
using PTManagementSystem.Presentation.DTOs;

namespace PTManagementSystem.Domain.Events
{
    public class PaymentInfoSentEvent
    {
        public string userId { get; }
        public PaymentInfoDto PaymentInfo { get; }

        public PaymentInfoSentEvent(string userId, PaymentInfoDto paymentInfo)
        {
            userId = userId;
            PaymentInfo = paymentInfo;
        }
    }
}

File: payment/paymentDemo/src/main/java/com/example/paymentDemo/event/PaymentInfoSentEvent.java
package com.example.paymentDemo.event;
import com.example.paymentDemo.dto.PaymentInfoDto;

public class PaymentInfoSentEvent {
    private String userId;
    private PaymentInfoDto paymentInfo;

    public PaymentInfoSentEvent() {
    }

    public PaymentInfoSentEvent(String userId, PaymentInfoDto paymentInfo) {
        this.userId = userId;
        this.paymentInfo = paymentInfo;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public PaymentInfoDto getPaymentInfo() {
        return paymentInfo;
    }

    public void setPaymentInfo(PaymentInfoDto paymentInfo) {
        this.paymentInfo = paymentInfo;
    }
}